mem-12-q.xml

```diff
@@ -85,7 +85,7 @@
       <column name="notes_de"></column>
       <column name="notes_fa"></column>
       <column name="notes_sv"></column>
-      <column name="notes_ru"></column>
+      <column name="notes_ru">Подобно слову {QaQ:v} данный глагол используется в контексте обозначения качества. Не используется для обозначения "не быть добрым"</column>
```

Comment:
Is this note necessary? Generally, the notes should be translations of each other.

Reply:
Yes. It is necessary. In entry `QaQ` we have similar note. I just made another one